### PR TITLE
Libunwind/1.5.0

### PR DESCRIPTION
**Pull request by arlyon**
Specify library name and version:  **libunwind/1.5.0**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by arlyon** [comment]
Of note, the tag name is v1.5 for some reason, while the other releases include the minor version. Not sure what the implications are, I assume it won't change.

**Review by Croydon — DISMISSED** [review]

File: recipes/libunwind/config.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 versions:
   "1.3.1":
     folder: all
+  "1.5.0":
+    folder: all
```
Comment (ericLemanissier) [inline recipes/libunwind/config.yml:5]:
```suggestion
    folder: all

```

**Review by Croydon — APPROVED** [review]

**Review by madebr — APPROVED** [review]

**Comment by madebr** [comment]
@arlyon Write a little message in #1 (with a reference to this pr) and you will get accepted in no-time.

**Review by uilianries — APPROVED** [review]

**Comment by arlyon** [comment]
Done, I appreciate the snappy responses here

**Comment by arlyon** [comment]
Just triggering CI

**Pull request merged**